raven/processes/wps_graph_single_hydrograph.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Mar 26 14:31:29 2019
+
+@author: ets
+"""
+
+from pywps import ComplexInput, ComplexOutput
+from pywps import Format
+from pywps import FORMATS
+from pywps import Process
+
+from pathlib import Path
+from raven.utilities.graphs import mean_annual_hydrograph, hydrograph, spaghetti_annual_hydrograph
+from matplotlib import pyplot as plt
+
+class GraphSingleHydrographProcess(Process):
+    def __init__(self):
+        inputs = [ComplexInput('sim', 'Stream flow simulations NetCDF File',
+                               abstract='Stream flow simulation time series',
+                               supported_formats=[FORMATS.NETCDF]),
+                  ]
+
+        outputs = [ComplexOutput('graph_ensemble_hydrographs', 'Figure showing the simple hydrographs of the included models.',
```

Comment:
Here we have only one model right ?

Reply:
Yes here there is only one model, single NetCDF file containing a Qsim and Qobs.

Reply:
Changed the "ensemble" to "single" for this case